app/android/src/main/java/com/capacitorjs/plugins/app/AppPlugin.java

```diff
@@ -73,7 +74,11 @@ public void exitApp(PluginCall call) {
     public void getInfo(PluginCall call) {
         JSObject data = new JSObject();
         try {
-            PackageInfo pinfo = getContext().getPackageManager().getPackageInfo(getContext().getPackageName(), 0);
+            PackageInfo pinfo = InternalUtils.getPackageInfo(
+                getContext().getPackageManager(),
+                getContext().getPackageName(),
+                PackageManager.PackageInfoFlags.of(0)
+            );
```

Comment:
```suggestion
            PackageInfo pinfo = InternalUtils.getPackageInfo(
                getContext().getPackageManager(),
                getContext().getPackageName()
            );
```
the method expects a long, not the flags, but also the long is not needed if it's 0